test/regression/cases/file_to_blackhole_0ms_latency/experiment.yaml

```diff
@@ -31,3 +31,9 @@ checks:
       series: total_rss_bytes
       # The machine has 12GiB free.
       upper_bound: 1.2GiB
+
+  - name: lost_bytes
+    description: "Allowable bytes not polled by log Agent"
```

Comment:
```suggestion
    description: "Available bytes not polled by log Agent"
```

Reply:
Oh hmm.

Reply:
I guess "allowable" implies there is an expected amount of bytes lost, where "available" doesn't imply that.